website_product_brand/ChangeLog.txt

```diff
@@ -0,0 +1,6 @@
+===================================================================
+Maintaining the changelog of the module for each commit.
+===================================================================
+
+1.0 : Serpent Consulting Services Pvt Ltd.
```

Comment:
We keep it as usual changelog. isnt it ok to have? Its just a history of commit with the reasons , saying which version contains what!